.chloggen/multi-instrumentation-command-flag.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: change multi instrumentation feature gate into command line flag
```

Comment:
could you please put here the old feature flag a new cmd flag or put it into the subtext